.github/actions/export-user-guides.sh

```diff
@@ -7,15 +7,26 @@
 
 OUTDIR=user-guides
 
-cd $GITHUB_WORKSPACE
+if [[ -z "$GITHUB_WORKSPACE" ]]
+then
+  echo "env var GITHUB_WORKSPACE is unset or does not exist"
```

Comment:
```suggestion
  echo "env var GITHUB_WORKSPACE is unset or does not exist"
  exit 1
```
the `|| exit` below will exit if the folder doesn't exist, but it the script will still proceed if `GITHUB_WORKSPACE` is unset

Reply:
Just tested it that way yes you're correct. Happy to commit that suggestion.